x-pack/plugins/data_visualizer/public/application/common/components/top_values/top_values.tsx

```diff
@@ -53,70 +52,48 @@ export const TopValues: FC<Props> = ({ stats, fieldFormat, barColor, compressed,
   const { fieldFormats } = data;
 
   if (stats === undefined || !stats.topValues) return null;
-  const {
-    topValues,
-    topValuesSampleSize,
-    count,
-    isTopValuesSampled,
-    fieldName,
-    sampleCount,
-    topValuesSamplerShardSize,
-  } = stats;
-
-  const totalDocuments = stats.totalDocuments;
+  const { topValues, fieldName, sampleCount } = stats;
 
-  const progressBarMax = isTopValuesSampled === true ? topValuesSampleSize : count;
+  const totalDocuments = stats.totalDocuments ?? 0;
+  const topValuesOtherCountPercent =
+    1 - (topValues ? topValues.reduce((acc, bucket) => acc + bucket.percent, 0) : 0);
+  const topValuesOtherCount = Math.floor(topValuesOtherCountPercent * (sampleCount || 0));
```

Comment:
although it doesn't matter logically here, for consistency it might be better to still use `??` because the point of the check is to look for `undefined`

Reply:
Fixed here [`80872e0` (#144646)](https://github.com/elastic/kibana/pull/144646/commits/80872e03e0de597573a005988aa03417e83cfb5a)